tests/fixtures/test-etcd/Dockerfile

```diff
@@ -3,11 +3,10 @@ FROM alpine:3.1
 # install common packages
 RUN apk add --update-cache curl tar && rm -rf /var/cache/apk/*
 
-ENV ETCD_VERSION v0.4.9
+# install etcdctl
+ENV ETCD_VERSION=2.0.13
+RUN curl -sSL https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz | \
+    tar zxv -C /usr/local/bin && ln -s /usr/local/bin/etcd-v${ETCD_VERSION}-linux-amd64/etcd* /usr/local/bin
 
-# install etcd and etcdctl
-RUN curl -sSL https://github.com/coreos/etcd/releases/download/$ETCD_VERSION/etcd-$ETCD_VERSION-linux-amd64.tar.gz \
-  | tar -vxz -C /usr/local/bin --strip=1
```

Comment:
You can extract things directly without using symlinks as long as you install alpine's `tar` package, which is more full-featured than the built-in busybox version.